src/index.js

```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
```

Comment:
- [  ]OPTIONAL  still has some files to be removed like setup, report, and app.test.js 
<img width="216" alt="Screenshot 2023-05-02 at 22 39 18" src="https://user-images.githubusercontent.com/61790587/235792328-eacbb0cc-8625-4643-8ad7-93ee04735fbe.png">